### PR TITLE
Policy Server security configuration.

### LINKED ISSUE #191: Allow the helm chart to configure the `securityContext` of the `default` Policy Server

### PR DESCRIPTION
**Pull request by jvanz**
## Description

Updates the Helm chart to make use of the latest changes in the policy server CRDs allowing users to define the "securityContext" of the workload. Therefore, this change allows users to customize the policy server "securityContext" changing the values of the Helm chart.

Fix PR makes use of the changes in the [controller](https://github.com/kubewarden/kubewarden-controller/pull/385)

Fix #191

### CODE REVIEW

File: charts/kubewarden-defaults/chart-values.yaml [hunk 1]
```diff
@@ -62,6 +62,18 @@ policyServer:
...
+      runAsNonRoot: true
+    container:
+      readOnlyRootFilesystem: true
+      privileged: false
+      runAsNonRoot: true
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+          - "all"
+
```
Comment (flavio) [inline charts/kubewarden-defaults/chart-values.yaml:76]: I see these are the prior default values we set. I don't think we will change them anytime soon, but I would prefer to have them defined via code inside of the controller (see my comment on the other PR).

I would propose to leave this entire section commented, to be a guideline to the user in case he wants to change it. At the same, by being commented, the default values inside of the `.go` files would take precedence.

Long story short, I fear that if we change these values in the go code, but forget to update this section of the helm chart, the change won't be enforced to our users.

File: charts/kubewarden-defaults/values.yaml [hunk 2]
```diff
@@ -74,6 +74,18 @@ policyServer:
...
+    pod:
+      runAsNonRoot: true
+    container:
+      readOnlyRootFilesystem: true
+      privileged: false
+      runAsNonRoot: true
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+          - "all"
```
Comment (flavio) [inline charts/kubewarden-defaults/values.yaml:87]: same here

File: charts/kubewarden-defaults/chart-values.yaml (hunk 1 above)
Comment (viccuad) [reply charts/kubewarden-defaults/chart-values.yaml:76]: This is a good point, I agree with it.

Comment (jvanz) [reply charts/kubewarden-defaults/chart-values.yaml]: Done

**Review by flavio — APPROVED** [review]
LGTM, I pushed a small commit that fixes the indentation of the commits. It makes them more readable and also consistent with the style we already had
